modules/packages/manifests/zstandard.pp

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+class packages::zstandard (
+    Pattern[/^v\d+\.\d+\.\d+$/] $zstandard_version,
+    String                      $zstandard_sha256,
+) {
+
+    packages::macos_package_from_s3 { "zstd-${zstandard_version}.dmg":
+        private             => false,
+        os_version_specific => false,
+        type                => 'dmg',
+        checksum            => $zstandard_sha256,
```

Comment:
dmg(w/pkg inside) don't utilize hashes.  Only the bin type uses them to ensure the file isn't downloaded on every puppet run.

Reply:
thx. I removed the sha entries